internal/wire/validation.go

```diff
@@ -28,11 +28,14 @@ type ValidationError struct {
 }
 
 // Error implements error interface.
-func (v ValidationError) Error() string {
+func (v *ValidationError) Error() string {
 	return v.err.Error()
 }
 
 // newValidationError returns new ValidationError.
+//
+// TODO remote and make callers use validateValue only?
```

Comment:
This TODO looks a bit unclear, should we clarify it, or fix it, or something?

Reply:
The linked issue is also not clear and "not ready". The whole validation here looks sus

Reply:
Ah, ok then!